src/gui/synthesispopover.cpp

```diff
@@ -1262,6 +1254,14 @@ void SynthesisPopover::onOpenMiscellaneousMenu(bool checked) {
     connect(helpAction, &QWidgetAction::triggered, this, &SynthesisPopover::onDisplayHelp);
     menu->addAction(helpAction);
 
+    // Send feedbacks
+    QWidgetAction *feedbacksAction = new QWidgetAction(this);
+    MenuItemWidget *feedbacksMenuItemWidget = new MenuItemWidget(tr("Send feedbacks"));
```

Comment:
```suggestion
    auto feedbacksMenuItemWidget = new MenuItemWidget(tr("Send feedbacks"));
```

Reply:
Done d100d681f08c906c7e42fae7a357f693961ec2f0

Reply:
Done in d100d681f08c906c7e42fae7a357f693961ec2f0